From 3e61d2524ab16f9db9ac15e5fc839892eec74cf4 Mon Sep 17 00:00:00 2001
From: k-dominik <k-dominik@users.noreply.github.com>
Date: Tue, 7 Dec 2021 21:48:30 +0100
Subject: [PATCH] fix collections.Iterable deprecation

---
 vigranumpy/lib/arraytypes.py | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/vigranumpy/lib/arraytypes.py b/vigranumpy/lib/arraytypes.py
index 84a55bf..f82a70e 100644
--- a/vigranumpy/lib/arraytypes.py
+++ b/vigranumpy/lib/arraytypes.py
@@ -39,7 +39,6 @@ import sys
 import copy
 import numpy
 import vigra.ufunc as ufunc
-import collections
 import vigra.vigranumpycore as vigranumpycore
 
 from vigra.vigranumpycore import AxisType, AxisInfo, AxisTags
@@ -48,6 +47,13 @@ if sys.version_info[0] > 2:
     buffer = memoryview
     xrange = range
 
+try:
+    from collections.abc import Iterable
+except ImportError:
+    # Python 2.7
+    from collections import Iterable
+
+
 def _preserve_doc(f):
     npy_doc = eval('numpy.ndarray.%s.__doc__' % f.__name__)
     f.__doc__ =  ("" if npy_doc is None else npy_doc) + \
@@ -1262,7 +1268,7 @@ class VigraArray(numpy.ndarray):
         try:
             res = numpy.ndarray.__getitem__(self, index)
         except:
-            if not isinstance(index, collections.Iterable):
+            if not isinstance(index, Iterable):
                 raise
             #create temporary index without AxisInfor in order to use np.ndarray.__getitem__
             tmpindex = [None if isinstance(x, AxisInfo) else x for x in index]
-- 
2.25.1

