From 22d5441e15f1e85f479f414086d89673921889bf Mon Sep 17 00:00:00 2001
From: k-dominik <k-dominik@users.noreply.github.com>
Date: Tue, 7 Dec 2021 21:49:49 +0100
Subject: [PATCH] fix numpy>1.20 build

---
 vigranumpy/lib/arraytypes.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/vigranumpy/lib/arraytypes.py b/vigranumpy/lib/arraytypes.py
index f82a70e..22fc6ad 100644
--- a/vigranumpy/lib/arraytypes.py
+++ b/vigranumpy/lib/arraytypes.py
@@ -289,6 +289,8 @@ AxisTags.fromJSON.__doc__ = _AxisTags_fromJSON.__doc__
 _constructArrayFromAxistags = vigranumpycore.constructArrayFromAxistags
 
 def _constructArrayFromOrder(cls, shape, dtype, order, init):
+    if hasattr(dtype, "type"):
+        dtype = dtype.type
     axistags = VigraArray.defaultAxistags(len(shape), order)
     return _constructArrayFromAxistags(cls, shape, dtype, axistags, init)
 
-- 
2.25.1

